### PR TITLE
Show command to change topic name on readme

### PR DESCRIPTION
**Pull request by asukiaaa**
I want to you to show command to change topic name like teleop_twist_keyboard of ros1.
https://github.com/ros-teleop/teleop_twist_keyboard

Thank you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,15 @@ Generic Keyboard Teleoperation for ROS
 
 ## Launch
```
Comment (clalancette) [inline README.md]:
```suggestion
## Run
```

File: README.md [hunk 2]
```diff
@@ -3,7 +3,15 @@ Generic Keyboard Teleoperation for ROS
...
 
-To run: `ros2 run teleop_twist_keyboard teleop_twist_keyboard`
+Run.
```
Comment (clalancette) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -3,7 +3,15 @@ Generic Keyboard Teleoperation for ROS
...
+```sh
+ros2 run teleop_twist_keyboard teleop_twist_keyboard
+```
+
+Publishing to a different topic (in this case my_cmd_vel).
```
Comment (clalancette) [inline README.md]:
```suggestion
Publishing to a different topic (in this case `my_cmd_vel`).
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by asukiaaa** [comment]
Thank you for the feedback.
I updated so.

**Review by clalancette — APPROVED** [review]
Thanks for the update!

**Pull request merged**